int32[3] range_reading
float32[3] lux_reading
